pkg/sql/sqlbase/column_type_encoding.go

```diff
@@ -93,6 +94,16 @@ func EncodeTableKey(b []byte, val tree.Datum, dir encoding.Direction) ([]byte, e
 			return encoding.EncodeStringAscending(b, string(*t)), nil
 		}
 		return encoding.EncodeStringDescending(b, string(*t)), nil
+	case *tree.DGeography:
```

Comment:
Is it likely that if we add forward indexing for geo types that their encodings will look like this?

Reply:
yes, if we decide with primary keys it will almost certainly be something like this.